environment.yml

```diff
@@ -0,0 +1,89 @@
+name: doccano
+channels:
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=main
+  - ca-certificates=2019.5.15=0
+  - certifi=2019.6.16=py37_0
+  - libedit=3.1.20181209=hc058e9b_0
+  - libffi=3.2.1=hd88cf55_4
+  - libgcc-ng=9.1.0=hdf63c60_0
+  - libstdcxx-ng=9.1.0=hdf63c60_0
+  - ncurses=6.1=he6710b0_1
+  - openssl=1.1.1c=h7b6447c_1
+  - pip=19.1.1=py37_0
+  - python=3.7.3=h0371630_0
+  - readline=7.0=h7b6447c_5
+  - setuptools=41.0.1=py37_0
+  - sqlite=3.28.0=h7b6447c_0
+  - tk=8.6.8=hbc83047_0
+  - wheel=0.33.4=py37_0
+  - xz=5.2.4=h14c3975_4
+  - zlib=1.2.11=h7b6447c_3
+  - pip:
+    - apache-libcloud==2.4.0
+    - applicationinsights==0.11.7
+    - asn1crypto==0.24.0
+    - brotli==1.0.7
+    - cffi==1.12.3
+    - chardet==3.0.4
+    - coverage==4.5.3
+    - cryptography==2.7
+    - defusedxml==0.6.0
+    - dj-database-url==0.5.0
+    - django==2.1.7
+    - django-cloud-browser==0.5.0
+    - django-filter==2.0.0
+    - django-heroku==0.3.1
+    - django-polymorphic==2.0.3
+    - django-rest-polymorphic==0.1.8
+    - django-webpack-loader==0.6.0
+    - django-widget-tweaks==1.4.2
+    - djangorestframework==3.8.2
+    - djangorestframework-csv==2.1.0
+    - djangorestframework-filters==0.10.2
+    - djangorestframework-xml==1.4.0
+    - environs==4.1.0
+    - faker==0.8.8
+    - flake8==3.6.0
+    - furl==2.0.0
+    - gunicorn==19.9.0
+    - idna==2.8
+    - lockfile==0.12.2
+    - marshmallow==2.19.5
+    - mccabe==0.6.1
+    - mixer==6.1.3
+    - model-mommy==1.6.0
+    - multidict==4.5.2
+    - numpy==1.16.4
+    - oauthlib==3.0.2
+    - orderedmultidict==1.0
+    - psycopg2==2.8.3
+    - psycopg2-binary==2.7.7
+    - pycodestyle==2.4.0
+    - pycparser==2.19
+    - pyflakes==2.0.0
+    - pyjwt==1.7.1
+    - python-dateutil==2.7.3
+    - python-dotenv==0.10.3
+    - python3-openid==3.1.0
+    - pytz==2018.4
+    - pyyaml==5.1.1
+    - requests==2.21.0
+    - requests-oauthlib==1.2.0
+    - seqeval==0.0.6
+    - six==1.11.0
+    - social-auth-app-django==3.1.0
+    - social-auth-core==3.0.0
+    - text-unidecode==1.2
+    - tornado==5.0.2
+    - unicodecsv==0.14.1
+    - unittest-xml-reporting==2.5.1
+    - urllib3==1.24.3
+    - vcrpy==2.0.1
+    - vcrpy-unittest==0.1.7
+    - whitenoise==4.1.2
+    - wrapt==1.11.2
+    - yarl==1.3.0
+prefix: /home/hakan/anaconda3/envs/doccano
```

Comment:
This looks like a line that shouldn't be checked in since it'll be different for each developer. Is there any way to make this generic?

Reply:
Hi. As I read from here https://stackoverflow.com/questions/41274007/anaconda-export-environment-file conda does not care about that line but puts it there anyway while exporting. I removed the line and built the environment without any issues.